extensions/kinesis/src/test/java/com/hazelcast/jet/kinesis/KinesisFailureTest.java

```diff
@@ -68,6 +69,8 @@ public class KinesisFailureTest extends AbstractKinesisTest {
     private static ContainerProxy PROXY;
     private static KinesisTestHelper HELPER;
 
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(KinesisFailureTest.class);
```

Comment:
Hmm, I think we can remove Hazelcast's Logger import and make KinesisTestHelper build it's own logger instead - I don't see much benefit in current solution of passing logger to KTH instead of building it there. Then we can just import slf4j's logger here.

Reply:
done: https://github.com/hazelcast/hazelcast/pull/23230/commits/569570162505ea1c7508ad083251f0d15eb91d11